modules/nw-cno-view.adoc

```diff
@@ -33,15 +33,15 @@ Spec: <1>
   Cluster Network:
     Cidr:         10.128.0.0/14
     Host Prefix:  23
-  Network Type:   OpenShiftSDN
+  Network Type:   OVN-Kubernetes
```

Comment:
"OVN-Kubernetes" is the user-friendly version of the name, but in the API it is _always_ `OVNKubernetes` with no hyphen. (Basically, anywhere you see `OpenShiftSDN` with no space, you probably want `OVNKubernetes` with no hyphen too.)